gpflow/likelihoods/heteroskedastic.py

```diff
@@ -25,7 +25,7 @@
 from ..utilities import positive
 
 
-class HeteroskedasticGaussianLikelihood(HeteroskedasticTFPConditional):
+class ParameterisedGaussianLikelihood(HeteroskedasticTFPConditional):
```

Comment:
Can this class be deleted? It looks like all it adds to the `HeteroskedasticTFPConditional` is the Gaussian distribution.

Reply:
Yes as it stands, though perhaps this can become the class which inherits from MultiLatentTFPConditional and permits dependence on both X and f?

Reply:
I think the `MultiLatentTFPConditional` already provides that behaviour. It is the `HeteroskedasticTFPConditional` which restricts it to the Xs.